oioswift/common/middleware/verb_acl.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (C) 2018 OpenIO SAS
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from swift.common.utils import get_remote_client, get_logger
+from swift.common.swob import Request, HTTPForbidden
+
+
+class VerbAclMiddleware(object):
+    """
+    Middleware that manages access (permission) to the methods
+    based on the IP address.
+    """
+
+    def __init__(self, app, conf):
+        self.app = app
+        self.logger = get_logger(conf)
+
+        self.verb_acl = {}
+        verb_acl = conf.get('verb_acl', None)
+        if not verb_acl:
+            raise ValueError('verb_acl: Not initialized in the config file.')
```

Comment:
Minor: position is not checked. We should provide an helper to check position, at least, please add a `TODO`